components/Alfabet.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { useState } from 'react'
+import { View, Vibration } from 'react-native'
+
+import styled from 'styled-components/native'
```

Comment:
Once again, this package shouldn't be separated from the others by one empty line - they are all outer packages, so they should stay together